### ISSUE TITLE
switch to the okcomputer gem for app status checking

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/argo#801]
DLSS is moving away from [is_it_working](https://github.com/tribune/is_it_working) for status checking/dependency enumeration (or as argo uses, the [cbeer fork](https://github.com/cbeer/is_it_working)).  switch over to our new default choice for status/dependency info, [okcomputer](https://github.com/sportngin/okcomputer).

**Comment by ndushay** [comment]
Steps:
- [ ] PR merged to master (#711)
- [ ] deploy
  - [ ] dev
  - [ ] stage-a, stage-b
  - [ ] prod-a, prod-b
- [ ] update Load Balancer (SAR ticket)
- [ ] update nagios 
  - [ ] basic checks
    - [ ] dev
    - [ ] stage-a, stage-b
    - [ ] prod-a, prod-b
    - [ ] argo-lb
- [ ] remove is_it_working
  - [ ] deploy
- [ ] update DevOpsDocs (new nagios, load balancer ...)

**Comment by cbeer** [comment]
Is it possible to simplify these steps by removing `is_it_working` and mounting okcomputer in its place at `/is_it_working` (in parallel with wherever you want it to end up):

```
  # API compatible with is_it_working checks
  match "/is_it_working" => "ok_computer/ok_computer#index", via: [:get, :options]
```

**Comment by ndushay** [comment]
understood, @cbeer  but would rather switch to something mounted at `/status`.

**Comment by ndushay** [comment]
it's also a good excuse to make sure we're on new nagios and have sane LB checks.

**Comment by cbeer** [comment]
That's fine -- I don't think that line precludes also mounting it at status, does it? I'm just suggesting we might try to avoid having two different status monitoring things running at once.

**Comment by ndushay** [comment]
this was a good idea, but it turns out mount_at is an attribute, so I think it can only have a single setting.

**Comment by ndushay** [comment]
I pushed up a branch, `okcomputer` that has my first attempt.  I deployed it at one point to -dev and -stage, but the rubydora check failed.  I am not working on it at this time.

The branch has a large number of "optional" checks - this is based on discussion with Ben Albritton and @jmartin-sul.   Require few, keep many for informational purposes - that was where we ended up.

**Comment by cbeer** [comment]
Fixed by #872.
